File: CJY/BJ/No_2775.java
import java.io.*;

public class No_2775 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int T = Integer.parseInt(br.readLine());

        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        for (int i = 0; i < T; i++) {
            int k = Integer.parseInt(br.readLine());
            int n = Integer.parseInt(br.readLine());

            int[] arr = new int[n];
            for (int j = 1; j <= n; j++) arr[j - 1] = j;
            for (int j = 0; j < k; j++) for (int l = 1; l < n; l++) arr[l] = arr[l - 1] + arr[l];
            bw.write(String.valueOf(arr[n - 1]));
            bw.newLine();
        }
        br.close();
        bw.flush();
        bw.close();
    }
}

File: CJY/BJ/No_2775.js
const rl = require("readline").createInterface({input: process.stdin, output: process.stdout});
let T = 0, k, n;
let answer = "";

rl.on("line", (s) => {
    if(T > 0) {
        if(T % 2 === 0) {
            k = s;
        } else {
            n = s;
            let arr = Array.from({length: n}, (v, i) => 1 + i++);
            for (let j = 1; j <= n; j++) arr[j - 1] = j;
            for (let j = 0; j < k; j++) for (let l = 1; l < n; l++) arr[l] = arr[l - 1] + arr[l];
            answer += arr[n - 1] + "\n";
        }
        T--;
        if(T === 0) {
            console.log(answer);
            rl.close();
        }
    } else {
        T = parseInt(s) * 2;
    }
}).on("close", () => {process.exit();});